config-client-polling/src/main/java/com/microsoft/sample/Application.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.sample;
```

Comment:
Add license header to all java files like [HERE](https://github.com/Azure-Samples/Azure-Spring-Cloud-Samples/blob/master/custom-config-server-client/src/main/java/com/microsoft/sample/customConfigServerClient/HelloController.java#L1-L5)

Reply:
fixed